app/core/dependencies.py

```
@@ -1,9 +1,19 @@
 from functools import lru_cache
+import logging
+from typing import Optional, Dict, Any, List
```

Ruff (0.11.9)

`typing.Dict` imported but unused

Remove unused import

(F401)